src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonOffenderEventListenerTest.kt

```diff
@@ -68,6 +68,17 @@ class PrisonOffenderEventListenerTest {
     }
   }
 
+  @Test
+  fun `process prisoner updated message`(): Unit = runBlocking {
```

Comment:
Could also add a test to `PrisonOffenderEventListenerIntTest` for this? That would validate the SQS listener (running in Localstack) and database is updated